webpack/containers/Application/Routes.js

```diff
@@ -1,22 +1,11 @@
 import React from 'react';
 import { Route } from 'react-router-dom';
 import { links } from './config';
-import Header from './Headers';
 
 export default () => (
   <div>
-    {links.map(({ path, component, text }) => {
-      const Page = component;
-      const withHeader = () => (
```

Comment:
on every route change `withHeader`  will recreate, it should exist as a separate component.